pkg/views/workspace/list/view.go

```diff
@@ -163,6 +168,9 @@ func getWorkspaceTableRowData(workspace serverapiclient.Workspace, specifyGitPro
 	if workspace.Projects != nil && len(workspace.Projects) > 0 && workspace.Projects[0].Repository != nil {
 		rowData.Repository = getRepositorySlugFromUrl(*workspace.Projects[0].Repository.Url, specifyGitProviders)
 	}
+	if workspace.Projects[0].Repository.Branch != nil {
```

Comment:
You need to move this to the `if` above.

On this line, `Repository` is not guaranteed to be defined and a nil-pointer exception might occur.